compiler/src/dmd/statementsem.d

```diff
@@ -2435,6 +2435,9 @@ package (dmd) extern (C++) final class StatementSemanticVisitor : Visitor
             */
             if (e.op == EXP.variable)
             {
+                // @@@DEPRECATION 2.102
```

Comment:
Just a nit, specify the version when it should be turned into an error.  It's easier to think about what needs moving along when you don't have to count back 10 versions from the current release.